### DISCUSSION TITLE
2024-04-07 온라인 회의

### DISCUSSION THREAD

**Discussion by jundoopop** [Dev-On-Play/spring-project#27] [온라인 회의록] [answered]
# 오늘 회의 목표

- [x]  지금까지 뭘 했는지 서로 정함

- [x]  다음주는 뭐 해갈지

# 공유 사항

### 현재 로컬에서 DB와 테스트를 진행 중

- 개발 단계에서 외부에서 접근 가능한 호스트/서버 를 하나 구축하여 DB 통합적으로 테스트 추진

### ticket 만들어서 진행하기

- 상세하고 용이한 개발 진행 상황 확인을 위함 

# FE

*깃허브에 역할별 티켓 올려서 개발 상황 공유*

> 다음주까지 기본적인 레이아웃 완성?

## 마이페이지

- 닉네임 편집 기능 shadcn 컴포넌트로 UI 만들었음

- 다음주까지 더 고도화
 
## 모각코 상세 페이지

- 현재는 UI 구성을 확인하기 위해 shadcn 컴포넌트를 가져와 배치했음

- 리액트 컴포넌트로서 더 기능하도록 개발 진행

## 메인페이지

- 메인페이지 각 요소에 배치할 react 컴포넌트 생각하고 구현하려고 했음

- 검색창, 카테고리, 카드뷰 컴포넌트로 만들기 (임시)

# BE

## 현황

- 현재 각 로컬 DB 사용 중
- 통합 DB 쓸 수 있도록 빠르면 다음주 이전에 Maria DB 구축

- DTO, 컨트롤러 명세서
- Open API로 path parameter에 따른 API 명세, uri 까지 볼 수 있도록 함

## 추후
- 명세서에 따라 개발??

# Github

## 브랜치 merge 관련

- develop 으로 merge: 한명이 컨펌하면 진행
- main 으로 merge: 세명이 컨펌하면 진행

## 티켓

- 티켓도 브랜치 따로 파서 만들 수 있음
- 브랜치 기능으로 개발 상황 공유 더 체계적으로 할 수 있음

**Comment by hdwhdwhdw** [comment, marked as answer]
이상 없습니다!
